.github/workflows/ci.yml

```diff
@@ -38,6 +38,7 @@ jobs:
         scala:
           - '2.12.17'
           - '2.13.10'
+          - '3.2.1'
```

Comment:
This seemed overly optimistic :laughing: We need to get rid or rewrtite the macros for it to work unfortunately :/